README.md

```diff
@@ -161,6 +163,8 @@ export interface CalendarProps<T extends ICalendarEventBase> {
 | `onSwipeEnd`                  | no       | `(date: Date) => void`                                            | Event handler which will be fired when swiping left or right. Memoize your function to prevent unnecesary re-renders                                                                                                                                                                                                                                                                                                                                              |
 | `swipeEnabled`                | no       | `boolean`                                                         |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | `showTime`                    | no       | `boolean`                                                         |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `minHour`                  | no       | `number`                                 | The first hour that will be displayed in week/day modes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `maxHour`                  | no       | `number`                                 | The final hour that will be displayed in week/day modes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Thanks,

If you provide examples in the storybook, it's helpful (not mandatory though)